src/HDInsight/HDInsight/help/Get-AzHDInsightHost.md

```diff
@@ -0,0 +1,146 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.HDInsight.dll-Help.xml
+Module Name: Az.HDInsight
+online version:
```

Comment:
please add online version

Reply:
Added

---
In reply to: [438670540](https://github.com/Azure/azure-powershell/pull/12124#discussion_r438670540) [](ancestors = 438670540)